docs/pages/application-access/getting-started.mdx

```diff
@@ -9,85 +9,92 @@ videoBanner: 5Uwhp3IQMHY
+
+<Admonition type="tip" title="Not yet a Teleport user?">
+If you have not yet deployed the Auth Service and Proxy Service, you should follow one of our [getting started guides](../getting-started.mdx).
+</Admonition>
 
 ## Step 1/3. Start Grafana
 
 We've picked Grafana for this tutorial since it's very easy to run with zero
 configuration required. If you have another web application you'd like to
 expose, skip over to **Step 2**.
 
-Grafana can be launched in a [Docker container](https://grafana.com/docs/grafana/latest/installation/docker/)
-with a single command:
+Grafana can be launched in a Docker container with a single command:
 
 ```code
 $ docker run -d -p 3000:3000 grafana/grafana
 ```
 
 ## Step 2/3. Install and configure Teleport
 
-Download the latest version of Teleport for your platform from our
-[downloads page](https://goteleport.com/teleport/download).
-
-Teleport requires a valid TLS certificate to operate and can fetch one automatically
-using Let's Encrypt [ACME](https://letsencrypt.org/how-it-works/) protocol.
-
-We will assume that you have configured DNS records for `teleport.example.com`
-and `*.teleport.example.com` to point to the Teleport node.
-
-(!docs/pages/includes/permission-warning.mdx!)
-
-Let's generate a Teleport config with ACME enabled:
+### Generate a token
+A join token is required to authorize a Teleport Application Service agent to
+join the cluster. Generate a short-lived join token and save it, for example,
+in `/tmp/token` on your Teleport Application Service host:
 
 ```code
-$ sudo teleport configure --cluster-name=teleport.example.com --acme --acme-email=alice@example.com -o file
+$ tctl tokens add \
+    --type=app \
+    --app-name=grafana \
+    --app-uri=http://localhost:3000
 ```
 
-<Admonition
-  type="note"
-  title="Web Proxy Port"
->
-  Teleport uses [TLS-ALPN-01](https://letsencrypt.org/docs/challenge-types/#tls-alpn-01)
-  ACME challenge to validate certificate requests which only works on port `443`. Make sure your Teleport proxy is accessible on port `443` when using ACME for certificate management.
-</Admonition>
+### Start Teleport
+On the host where you will run the Teleport Application Service, download the latest version of Teleport for your platform from our
+[downloads page](https://goteleport.com/teleport/download).
 
 Now start Teleport and point it to the application endpoint:
 
 ```code
 $ sudo teleport start \
-  --roles=proxy,auth,app \
+  --roles=app \
   --app-name=grafana \
-  --app-uri=http://localhost:3000
+  --token=/tmp/token \
+  --app-uri=http://localhost:3000 \
+  --auth-server=https://teleport.example.com:3080
 ```
 
-Make sure to update `--app-name` and `--app-uri` accordingly if you're using
-your own web application.
+Change `https://teleport.example.com:3080` to the address and port of your Teleport Auth Server. If you are a Teleport Cloud cluster, use your tenant's subdomain, e.g., `mytenant.teleport.sh`. 
```

Comment:
Does app service support direct dial? If not, this might need to be changed to `the address and port of your Teleport Proxy Server`.

(My crystal ball tells me that allowing `--auth-server` to point to an auth server or a proxy server depending on whether or not you want to use IoT mode will be looked back on as our billion dollar mistake)